### ISSUE TITLE
Logic app Access Control triggers Template Should Not Contain Blanks 

### ISSUE THREAD

**Issue by LeonB87** [Azure/arm-ttk#167]
When setting a Azure Logic app workflow setting to "only other logic apps". An empty property must be defined. This causes the ARM-TTK rule "Template Should Not Contain Blanks" to trigger.

example ARM code that causes the rule to be triggered.

```json
"resources": [
    {
        "name": "[parameters('LogicAppName')]",
        "type": "Microsoft.Logic/workflows",
        "location": "[parameters('location')]",
        "tags": {},
        "apiVersion": "2019-05-01",
        "properties": {
            "definition": {},
            "parameters": {},
            "accessControl": {
                "triggers": {
                    "allowedCallerIpAddresses": []
                },
                "actions": {
                    "allowedCallerIpAddresses": []
                },
                "contents": {
                    "allowedCallerIpAddresses": []
                }
            }
        }
    }
]
```

description of the (arm)setting https://msftplayground.com/2020/02/managing-access-control-for-logic-apps/

**Comment by LeonB87** [comment]
Created PR that should fix this Issue.
Hope I've done it the correct way.

PR #168

**Comment by bmoore-msft** [comment]
fixed in #168
